apps/core0/builtin/power.go

```diff
@@ -1,31 +1,167 @@
 package builtin
 
+// #include <sys/syscall.h>
```

Comment:
@maxux 
This builds fine locally, and on builder but fails on travis with this error. Any idea ?
```
# github.com/threefoldtech/0-core/apps/core0/builtin
apps/core0/builtin/power.go:147:3: could not determine kind of name for C.KEXEC_FILE_NO_INITRAMFS
apps/core0/builtin/power.go:142:3: could not determine kind of name for C.SYS_kexec_file_load
gcc errors for preamble:
In file included from apps/core0/builtin/power.go:4:0:
/usr/include/linux/kexec.h:40:2: error: unknown type name 'size_t'
  size_t bufsz;
  ^
/usr/include/linux/kexec.h:42:2: error: unknown type name 'size_t'
  size_t memsz;
  ^
/usr/include/linux/kexec.h:49:31: error: unknown type name 'size_t'
 extern int kexec_load(void *, size_t, struct kexec_segment *,
                               ^
The command "bash test.sh" exited with 2.
```